.github/workflows/infra.yml

```diff
@@ -23,6 +23,7 @@ jobs:
     - run: npm ci
     - run: npm run build
     - run: npx playwright install-deps
+    - run: npx playwright install
```

Comment:
```suggestion
    - run: npx playwright install --with-deps
```
nit: remove the file above it.

Reply:
I actually want to see groups for those in the UI!